spec/controllers/reservations_controller_spec.rb

```diff
@@ -29,8 +30,45 @@
     end
   end
 
+  describe "DELETE #destroy" do
+    context "when user is logged in" do
+      before do
+        sign_in_user
+        delete_destroy(wishlist.id, item.id, reservation.id)
```

Comment:
I'm not 100% sure, but isn't this part of the docs talking about that?
(taken from **Associations** part [http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Using_factories](from here)):

> It's possible to set up associations within factories. If the factory name is the same as the association name, the factory name can be left out.
```ruby
factory :post do
  # ...
  author
end
```